extensions/wikia/GlobalNavigation/scripts/GlobalNavigationNotifications.js

```diff
@@ -1,3 +1,8 @@
+// TODO XW-1943
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS1135)